### ISSUE TITLE
When running INSET_test.py, bert-base-uncased cannot be found

### ISSUE THREAD

**Issue by forest1988** [dreasysnail/INSET#2]
Thank you for sharing your code and trained model.

I ran INSET_test.py using the README.md file as a reference, but I got the following error about loading the BertModel.

```
(LSP)  $ python3 INSET_test.py
decode type:  beam
Model name 'bert-base-uncased' was not found in model name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese). We assumed 'https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-vocab.txt' was a path or url but couldn't find any file associated to this path or url.
Model name 'bert-base-uncased' was not found in model name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese). We assumed 'https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased.tar.gz' was a path or url but couldn't find any file associated to this path or url.
Traceback (most recent call last):
  File "INSET_test.py", line 54, in <module>
    model_bert = BertModel.from_pretrained('bert-base-uncased', state_dict=torch.load('models/BERT-pretrain-1-step-5000.pkl')).cuda()
AttributeError: 'NoneType' object has no attribute 'cuda'
```

To be sure, I checked the behavior of the version of "pytorch_pretrained_bert" in the conda virtual environment, and I got the same error.

```
(LSP)  $ ipython
Python 3.6.9 |Anaconda, Inc.| (default, Jul 30 2019, 19:07:31)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.8.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import torch

In [2]: from pytorch_pretrained_bert import *
Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.

In [3]: model = BertModel.from_pretrained('bert-base-uncased')
Model name 'bert-base-uncased' was not found in model name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese). We assumed 'https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased.tar.gz' was a path or url but couldn't find any file associated to this path or url.
```

Possibly, do I need to download "bert-base-uncased" from "pytorch_pretrained_bert" beforehand to get this code to work?
I would appreciate it if you know and could tell me how to avoid the error.

**Comment by forest1988** [comment]
Excuse me to bother you again.

I checked the source code of "pytorch_pretrained_bert" and estimated that the problem is occurring in cache_path() in file_util.py.
I got "FileExistsError".

I checked my home directory, and found there is a symbolic link named ".pytorch_pretrained_bert". However, the target of the link does not exist anymore.
It may have been caused when I switched pytorch_pretrained_bert to the newer version, which has another cache directory name. I think I've deleted the cached file, but have forgotten to delete the symbolic link.

I'm sorry to bother you with the problem caused by my environment management.

Now INSET_test.py works, and I got output.txt.
Thank you!
